ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx

```diff
@@ -26,7 +26,11 @@ const vulnerabilitiesWorkloadCveSinglePath = `${vulnerabilitiesWorkloadCvesPath}
 const vulnerabilitiesWorkloadCveImageSinglePath = `${vulnerabilitiesWorkloadCvesPath}/images/:imageId`;
 const vulnerabilitiesWorkloadCveDeploymentSinglePath = `${vulnerabilitiesWorkloadCvesPath}/deployments/:deploymentId`;
 
-function WorkloadCvesPage() {
+export type WorkloadCvesPageProps = {
+    pageTitle: string;
+};
```

Comment:
It was tempting to wrap this object in a Context for all of Workload CVEs, especially since more props are coming later, but the clarity of directly passing values outweighs the convenience of Context in this case to me.